From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: "Simental Magana, Marcos" <marcos.simental.magana@intel.com>
Date: Tue, 5 May 2015 13:08:47 -0500
Subject: [PATCH] Add read only sudoers.d dir

2020-07-28 update: continue using the old "#includedir" directive
instead of "@includedir" to ease downgrades or possibly incomplete
upgrades/downgrades.

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 plugins/sudoers/sudoers.in | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/plugins/sudoers/sudoers.in b/plugins/sudoers/sudoers.in
index 554d2e7..f247257 100644
--- a/plugins/sudoers/sudoers.in
+++ b/plugins/sudoers/sudoers.in
@@ -93,5 +93,7 @@ root ALL=(ALL) ALL
 # Defaults targetpw  # Ask for the password of the target user
 # ALL ALL=(ALL) ALL  # WARNING: only use this together with 'Defaults targetpw'
 
-## Read drop-in files from @sysconfdir@/sudoers.d
-@includedir @sysconfdir@/sudoers.d
+## Read drop-in files from /usr/share/defaults/sudo/sudoers.d and @sysconfdir@/sudoers.d
+## (the '#' here does not indicate a comment)
+#includedir /usr/share/defaults/sudo/sudoers.d
+#includedir @sysconfdir@/sudoers.d
